pkg/odo/cli/url/list.go

```diff
@@ -76,15 +76,15 @@ func (o *URLListOptions) Run() (err error) {
 
 			log.Infof("Found the following URLs for component %v", componentName)
 			tabWriterURL := tabwriter.NewWriter(os.Stdout, 5, 2, 3, ' ', tabwriter.TabIndent)
-			fmt.Fprintln(tabWriterURL, "NAME", "\t", "URL", "\t", "PORT")
+			fmt.Fprintln(tabWriterURL, "NAME", "\t", "URL", "\t", "PORT", "\t", "SECURE")
 
 			// are there changes between local and cluster states?
 			outOfSync := false
 			for _, i := range localUrls {
 				var present bool
 				for _, u := range urls.Items {
 					if i.Name == u.Name {
-						fmt.Fprintln(tabWriterURL, u.Name, "\t", url.GetURLString(url.GetProtocol(routev1.Route{}, u), "", u.Spec.Rules[0].Host), "\t", u.Spec.Rules[0].IngressRuleValue.HTTP.Paths[0].Backend.ServicePort.IntVal)
+						fmt.Fprintln(tabWriterURL, u.Name, "\t", url.GetURLString(url.GetProtocol(routev1.Route{}, u), "", u.Spec.Rules[0].Host), "\t", u.Spec.Rules[0].IngressRuleValue.HTTP.Paths[0].Backend.ServicePort.IntVal, "\t", u.Spec.TLS != nil)
```

Comment:
can we make this more readable? just top of my head maybe extract `url.GetURLString(url.GetProtocol(routev1.Route{}, u), "", u.Spec.Rules[0].Host)` into a helper?

Reply:
> url.GetURLString(url.GetProtocol(routev1.Route{}, u), "", u.Spec.Rules[0].Host)

But url.GetURLString() is already a helper function.

Reply:
I am ok if you feel we cannot make this more clean.

Reply:
I tried certain things but it was making things a bit complicated and weird for me.